src/main/java/org/elasticsearch/discovery/zen/ZenDiscovery.java

```diff
@@ -692,12 +692,10 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
 
     static class ProcessClusterState {
         final ClusterState clusterState;
-        final PublishClusterStateAction.NewClusterStateListener.NewStateProcessed newStateProcessed;
         volatile boolean processed;
 
```

Comment:
It's unrelated to this change: this parameter is also unused on master. I removed it because it confused me when trying to understand the code.